### PR TITLE
Add tent-css w/ npm auto-update via single package.json

### PR DESCRIPTION
**Pull request by sashberd**
Pull request for issue: #12248

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/sitetent/tentcss
 * Official website (optional, not the repository): https://css.sitetent.com/
 * NPM package url (optional): https://www.npmjs.com/package/tent-css
 * License and its reference: [MIT](https://github.com/sitetent/tentcss/blob/master/package.json#L7)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 16
   - Count of stars: 386
   - Count of forks: 26
 * NPM download stats (optional):
   - Downloads in the last day: 
   - Downloads in the last week: 18
   - Downloads in the last month: 

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/tent-css/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "sass",
+    "responsive",
+    "framework"
+  ],
+  "npmName": "tent-css",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "!(06_components){,**/*}"
```
Comment (MattIPv4) [inline ajax/libs/tent-css/package.json]: What is the intention of this glob?

Comment (xtuc) [reply ajax/libs/tent-css/package.json]: I don't think we support that

Comment (sashberd) [reply ajax/libs/tent-css/package.json]: In old versions there is folder in dist called 06_components
[https://github.com/sitetent/tentcss/tree/v1.1.1/dist](https://github.com/sitetent/tentcss/tree/v1.1.1/dist)

Comment (MattIPv4) [reply ajax/libs/tent-css/package.json]: I'm not sure glob syntax like this will work -- maybe best to add it to the .gitignore?

**Comment by sashberd** [comment]
@MattIPv4 Thanks. Sorry for delay, because of world wide situation, I do not have time now 😞

**Pull request merged**